boot/zephyr/main.c

```diff
@@ -73,7 +81,7 @@ void main(void)
 	SYS_LOG_INF("Setting vector table to %p", vt);
 
 	/* Not all targets set the VTOR, so just set it. */
-	_scs_relocate_vector_table((void *) vt);
+	relocate_vector_table((void *) vt);
```

Comment:
I can rebase the last 2 patches ontop of whatever fix you want to do for the _scs_relocate_vector_table issue.